socs/agents/pysmurf_controller/agent.py

```diff
@@ -1138,6 +1138,20 @@ def all_off(self, session, params):
 
             return True, "Everything off"
 
+    def restart_rssi(self, session):
```

Comment:
Could decorate with `@ocs_agent.param('_')`, which is for operations that don't accept any parameters.